### DISCUSSION TITLE
Copilot API request timeout.

### DISCUSSION THREAD

**Discussion by Wiblz** [community/community#7782] [Copilot Conversations] [answered]
No solutions are generated by Copilot. API requests are timed out: 
`[ERROR] [fetch] [2021-11-16T16:32:49.134Z] Unhandled status from server:,408,{"error":{"message":"upstream request timeout"}}`

**Comment by MulleDK19** [comment, marked as answer]
I've had this a couple of times in the past few hours. They might be over capacity.
